tests/valid/draft-template.pages.text

```diff
@@ -565,10 +565,10 @@ Internet-Draft              Abbreviated Title                 March 2009
    [I-D.narten-iana-considerations-rfc2434bis]
               Alvestrand, H. T. and T. Narten, "Guidelines for Writing
               an IANA Considerations Section in RFCs", Work in Progress,
-              Internet-Draft, draft-narten-iana-considerations-
-              rfc2434bis-09, March 26, 2008,
-              <https://www.ietf.org/archive/id/draft-narten-iana-
-              considerations-rfc2434bis-09.txt>.
+              Internet-Draft,
+              draft-narten-iana-considerations-rfc2434bis-09, March 26,
+              2008, <https://www.ietf.org/archive/id/draft-narten-iana-c
```

Comment:
See https://github.com/ietf-tools/xml2rfc/pull/965/#issuecomment-1413237933

Reply:
Yeah, this is ugly.